src/common/System.h

```diff
@@ -31,6 +31,10 @@ SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #ifndef COMMON_SYSTEM_H_
 #define COMMON_SYSTEM_H_
 
+extern const gameinfo_t* current_game;
```

Comment:
Why do you use current_game directly if you have a getter? I think this should be an implementation detal.